src/smokeTest/java/uk/gov/hmcts/reform/pip/rules/smoketests/SmokeTestConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.pip.rules.smoketests;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.PropertySource;
+
+@ComponentScan("uk.gov.hmcts.reform.pip.rules.smoketests")
+@PropertySource("application.properties")
+public class SmokeTestConfiguration {
```

Comment:
is this enough to make the pipeline pass without and actual test?

this on it's own looks fine, although the tests are intentionally not added here as we don't want people to forget to add smoke tests. but I do believe the pipeline should pass first time.

Any thoughts?

Reply:
it worked for the pip-business-rules, but didn't for hmi case hq, so might have spoken to soon.
I don't now Java, so this is a copy and paste job with a developers review.

If you don't think this is best then can close the PR

Reply:
can you request their review on this PR, I'll also ping a couple of others.

Reply:
Should we add a TODO to the Read me that team should add smoke tests based on functionality ? Just as a reminder for them.

Reply:
@adusumillipraveen I think the pipeline needs the smoke test to run. It doesn't need actually tests within it, but it does need setting up.
There can get the generic set up of smoke test, but then will need a update for the folder structure and naming.

Reply:
After correcting the folder paths, removing the flag that Tim mentioned and added some missing lines in the gradle build file it now works.
As well as this working in this project it also passes the build, which was failing in another project:
https://sds-build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_HMI%2Fhmi-case-hq-emulator/detail/master/25/pipeline

the user would just need to update naming same as anything else